scripts/build.ts

```diff
@@ -1,18 +1,18 @@
-import { toHashString } from 'https://deno.land/std@0.193.0/crypto/to_hash_string.ts';
+import { encodeHex } from 'https://deno.land/std@0.215.0/encoding/hex.ts';
 import {
   basename,
   extname,
   join,
-} from 'https://deno.land/std@0.193.0/path/posix.ts';
-import manifestJson from '../src/manifest.json' assert {
+} from 'https://deno.land/std@0.215.0/path/mod.ts';
+import manifestJson from '../src/manifest.json' with {
   type: 'json',
 };
-import manifestSchema from '../src/manifestSchemaAdjusted.json' assert {
+import manifestSchema from '../src/manifestSchemaAdjusted.json' with {
   type: 'json',
 };
-import { assertEquals } from 'https://deno.land/std@0.193.0/testing/asserts.ts';
+import { assertEquals } from 'https://deno.land/std@0.215.0/testing/asserts.ts';
 // Always require --allow-net https://github.com/denoland/deno_emit/issues/81
-import { bundle } from 'https://deno.land/x/emit@0.24.0/mod.ts';
+import { bundle } from 'https://deno.land/x/emit@0.35.0/mod.ts';
```

Comment:
🤷 I don't know why needed the updating dependency in build scrips...


Maybe emit? https://github.com/denoland/deno_emit/compare/0.24.0...0.35.0